.github/workflows/testing.yml

```diff
@@ -198,7 +198,7 @@ jobs:
         if: matrix.tarantool != 'brew' && matrix.tarantool != 'master'
 
```

Comment:
Tarantool's issue in the commit message is not linked by just `#xxxx`, because it is in another repository. I would give a full link instead.

Reply:
And, hm, title of the second commit is not relevant to what you're actually fixed there: it has nothing about openssl.

Reply:
Fixed all.